focus.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "FOCUS",
+  "aliases": [
+    "Fisheries Open Source Community Software"
+  ],
+  "description": "FOCUS is an open-source community that aims to oversee and maintain free software dedicated to the management and preservation of the fishery resource.",
+  "website": "http://www.focus.fish/about/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "http://www.focus.fish/"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 14,
+      "evidenceText": "All FOCUS deliverables are jointly produced and published as free and open-source. As such FOCUS truly offers its members an interoperable fisheries management platform in the most cost efficient and open way. Furthermore, FOCUS supports the 14:th of the UN Sustainable Development Goals",
+      "evidenceURL": "https://www.havochvatten.se/en/eu-and-international/international-cooperation/focus.html"
+    }
+  ],
+  "sectors": [
+    "Environment",
+    "Sustainability"
+  ],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/UnionVMS/UVMS-MobileTerminalModule-APP",
```

Comment:
@Msgl could you please justify how you settled for this [repo](https://github.com/UnionVMS/UVMS-MobileTerminalModule-APP) of all the `repos` under [UnionVMS](https://github.com/UnionVMS) Github org. Thanks.

Reply:
As I understand by going through their repository, many repos are linked to one another, and I could not find a clear structure where it shows that one repo is the "main" one. That was the reason why I initially added their GitHub URL without selecting a specific repo. 

I went through their manual installation documentation at https://focusfish.atlassian.net/wiki/spaces/UVMS/pages/15925281/Liquibase+setup, where a repo UVMS-MobileTerminalModule-DB is mentioned for installing and using the application on a mobile terminal. I searched through their repo and found the UVMS-MobileTerminalModule-APP which seems to have the same README.md file. So if I have understood correctly, and this is indeed the repo that a person can clone and use for mobile monitoring of fish resources, I believe this is the appropriate repo as it also provides usability for the user.

Reply:
@Msgl sounds convincing. Let's proceed with this then.